.github/workflows/docs.yaml

```
@@ -61,15 +61,13 @@ jobs:
       uses: actions/cache@v3.0.6
       with:
         path: ${{ steps.prepare-cache-pip.outputs.cache-dir }}
-        key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/setup.cfg') }}
+        key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/pyproject.toml') }}
         restore-keys: |
           ${{ runner.os }}-${{ matrix.python-version }}-pip-
 
-    - name: install requirements.txt
+    - name: install broadbean & docs dependancies
       run: |
-        pip install -r requirements.txt
-    - name: install broadbean
-      run: pip install .
+        pip install -c requirements.txt .[docs,test]
```

I think we should document this
```suggestion
        pip install -c requirements.txt .[docs,test]
        # currently we are building api docs of the whole package including tests so we need to be able to import the tests and therefore require test dependencies to build the docs. 
```

---

good call!